src/Controller/EnMarche/AdherentController.php

```diff
@@ -102,14 +103,14 @@ public function pinInterestsAction(Request $request): Response
      * @Method("GET|POST")
      * @Security("is_granted('CREATE_COMMITTEE')")
      */
-    public function createCommitteeAction(Request $request): Response
+    public function createCommitteeAction(Request $request, CommitteeCreationCommandHandler $commandHandler): Response
     {
         $command = CommitteeCreationCommand::createFromAdherent($user = $this->getUser());
         $form = $this->createForm(CreateCommitteeCommandType::class, $command);
         $form->handleRequest($request);
 
         if ($form->isSubmitted() && $form->isValid()) {
-            $this->get('app.committee.creation_handler')->handle($command);
+            $commandHandler->handle($command);
```

Comment:
to be consistent you should use `$this->get(CommitteeCreationCommandHandler::class)` here.

Reply:
ok, done